### ISSUE TITLE
Using kde2nix packages in NixOS config

### ISSUE THREAD

**Issue by kcalvelli** [nix-community/kde2nix#33]
I apologize for the beginner question:  how can I use kde2nix packages in my NixOS config with flakes?

**Comment by K900** [comment]
As the readme says,

> Add this repo to your flake inputs and include the NixOS module in your config, then `services.xserver.desktopManager.plasma6.enable = true`.

Is there a specific part of this you're having trouble with?

**Comment by kcalvelli** [comment]
I should have been more specific.  I have added the repo in flake inputs, included the NixOS module in my config, and am happily running plasma6.  The trouble is with the other packages that are not installed by default, such as Kate, Kmail, Merkuro, etc.

When I add them with environment.systemPackages, how can I add them from the kde2nix packages vs standard nixos packages?  I have tried importing inputs.kde2nix.packages, but have not been successful.
